src/api/is-device-supported.js

```diff
@@ -0,0 +1,15 @@
+import getDeviceSize from '../utils/get-device-size';
+
+
+/**
+ * Checks if the device is supported as the target for the `resize` API method.
+ * @function
+ * @async
+ * @name isDeviceSupportedAsResizeTarget
+ * @param {string} deviceName - Specifies the name of the target device. You can use the values specified in the Device Name column of [this table]{@link http://viewportsizes.com/}.
+ */
```

Comment:
The purpose of the function is to check programmatically if `deviceName` is in http://viewportsizes.com. It should be used to validate user input.